internal/services/policy/assignment_base_resource.go

```diff
@@ -326,6 +336,26 @@ func (br assignmentBaseResource) arguments(fields map[string]*pluginsdk.Schema)
 			},
 		},
 
+		"non_compliance_message": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"message": {
```

Comment:
as this is part of the block already should we make this
```suggestion
					"content": {
```

Reply:
Hi - thanks for looking at this.

I went for `message` as this matched the ARM REST schema:

https://docs.microsoft.com/en-us/azure/governance/policy/concepts/assignment-structure#non-compliance-messages

Happy to change it if you want but felt this was analogous to the source. Please let me know!

Reply:
we try to be more clear the accurate to the API as the resulting UX can be less than optimal having message.message is less clear and ambiguous then message.content 🙂

Reply:
Sure thing, will update now 👍

Reply:
Resolved